zim/plugins/base/imagegenerator.py

```
@@ -159,7 +160,11 @@ def set_from_generator(self, text, image_file):
 		# TODO: use index table to keep track and clean up when ref count is zero ?
 		self.data = text
 		self.image_file = self._new_image_file()
-		image_file.moveto(self.image_file)
+		try:
+			image_file.moveto(self.image_file)
+		except FileExistsError:
```

In what situation is this error raised? Would it be more appropriate to forcefully overwrite the file to ensure it is up-to-date rather than to fail silently?